aten/src/ATen/native/native_functions.yaml

```diff
@@ -2238,13 +2238,13 @@
 
 - func: eye.out(int n, *, Tensor(a!) out) -> Tensor(a!)
   dispatch:
-    CPU: eye_out_cpu
+    CPU, Meta: eye_out_cpu
     CUDA: eye_out_cuda
     MPS: eye_out_mps
 
 - func: eye.m_out(int n, int m, *, Tensor(a!) out) -> Tensor(a!)
   dispatch:
-    CPU: eye_out_cpu
+    CPU, Meta: eye_out_cpu
```

Comment:
(wasn't this split out into a separate PR somewhere? np though if you want to land it all together)

Reply:
yes, once I rebase past it this will go away